src/pip/_internal/cli/cmdoptions.py

```diff
@@ -953,7 +953,7 @@ def check_list_path_option(options: Values) -> None:
     metavar="feature",
     action="append",
     default=[],
-    choices=["2020-resolver", "fast-deps", "in-tree-build"],
+    choices=["2020-resolver", "fast-deps"],
```

Comment:
Out of curiosity, why is `--use-feature=2020-resolver` still a thing?

Reply:
But the 2020 reaolver is default, so why do we need a flag to *enable* it? Is it just so that people with the legacy resolver set as default in their config can override it?

It's not a big deal either way, I was just curious 🙂

Reply:
I guess? I didn't read what you wrote currently, and am currently feeling dum-dum.